libraries/AP_Scripting/AP_Scripting.cpp

```diff
@@ -236,6 +236,12 @@ void AP_Scripting::thread(void) {
         }
         delete lua;
 
+        // clear allocated i2c devices
+        for (uint8_t i=0; i<SCRIPTING_MAX_NUM_I2C_DEVICE; i++) {
+            delete _i2c_dev[i];
```

Comment:
can't this cause us to delete twice? do we need to set to nullptr after delete

Reply:
Your right.